app/controllers/verify/usps_controller.rb

```diff
@@ -11,7 +11,12 @@ def index
     def create
       create_user_event(:usps_mail_sent, current_user)
       idv_session.address_verification_mechanism = :usps
-      redirect_to verify_review_url
+
+      if current_user.decorate.needs_profile_usps_verification?
+        redirect_to account_path
```

Comment:
the flow just wasn't completed. I updated the specs and Code Climate should be happy now